docs/installation/post-install.rst

```diff
@@ -1,23 +1,23 @@
-Post-Installation Procedure
-===========================
+After Installation
+==================
+
+After bootstrapping the cluster, the Prometheus and AlertManager services used
+to monitor the system will not run (their respective :term:`Pods <Pod>` remain
+in the "Pending" state) until persistent storage is available.
 
 .. _Provision Prometheus Storage:
 
 Provision Storage for Prometheus Services
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-After bootstrapping the cluster, the Prometheus and AlertManager services used
-to monitor the system **will not be running** (the respective :term:`Pods
-<Pod>` will remain in *Pending* state), because they require persistent storage
-to be available.
-
-You can either provision these storage volumes on the :term:`Bootstrap
-node`, or later on other nodes joining the cluster. It is even recommended to
-separate :ref:`Bootstrap services <node-role-bootstrap>` from :ref:`Infra
-services <node-role-infra>`.
-
-To create the required *Volume* objects, write a YAML file with the following
-contents, replacing ``<node_name>`` with the name of the :term:`Node` on which
-to run Prometheus and AlertManager, and ``<device_path[2]>`` with the ``/dev``
+
+You can either provision storage volumes on the :term:`Bootstrap node`, or
+later on other nodes joining the cluster. Separating :ref:`Bootstrap services
+<node-role-bootstrap>` from :ref:`Infra services <node-role-infra>` is
+recommended.
+
+To create the required "Volume" objects, copy the following code to a YAML
```

Comment:
We should have some semantic role for object types... Just putting them between quotes doesn't feel right.